### PR TITLE
React router update for sequence files

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

What did you change in this pull request? Provide a description of files changed, user interactions changed, etc. Include how to test your changes.

## Related issue

Link to the GitHub issue this pull request addresses using the `#issuenum` format. If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
*   [ ] Tests added (or description of how to test) for any new features.
*   [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/webapp/resources/js/pages/sequence-files/components/FastQCCharts.jsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const StyledImage = styled.img`
 
 export default function FastQCCharts() {
   const { sequenceObjectId, fileId } = useParams();
+  console.log({ sequenceObjectId, fileId });
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/sequence-files/components/FastQCCharts.jsx:26]: Remove this logging statement

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]

File: src/main/webapp/resources/js/pages/sequence-files/index.js [hunk 2]
```diff
@@ -18,24 +17,26 @@ See: https://webpack.js.org/guides/public-path/#on-the-fly
...
 
+function App() {
+  const [path, route] = getRootPath(location.pathname);
+
+  return (
+    <Routes>
+      <Route path={path} element={<FastQC current={route} />}>
+        <Route path="overrepresented" element={<OverRepresentedSequences />} />
+        <Route path="details" element={<FastQCDetails />} />
+        <Route path="charts" element={<FastQCCharts />} />
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/sequence-files/index.js:28]: Lets combine these two (`<Route path="charts" element={<FastQCCharts />} />` and `<Route path="summary" element={<FastQCCharts />} />`) into `<Route path="*" element={<FastQCCharts />} />`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Just a couple of changes but otherwise good to go

File: src/main/webapp/resources/js/pages/sequence-files/index.js (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/sequence-files/index.js:33]: Cannot do that for the reason I told you yesterday where the `*` does not capture the `/` router

Comment (deepsidhu85) [reply src/main/webapp/resources/js/pages/sequence-files/index.js:33]: Whoops my bad. Thanks

Comment (deepsidhu85) [reply src/main/webapp/resources/js/pages/sequence-files/index.js]: 566b889ac70a7df4bc8cd8ea74c173441ac8cab6

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**